File: Level 0/java/HelloWorldwithToast/app/src/main/java/com/peace/helloworldwithtoast/MainActivity.java
package com.peace.helloworldwithtoast;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText et;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        et=findViewById(R.id.editText);
    }

    public void toastHello(View view) {
        Toast.makeText(this,"Hello "+et.getText(),Toast.LENGTH_SHORT).show();
    }
}

File: Level 0/kotlin/HelloWorldWithToastKotlin/app/src/main/java/com/peace/helloworldwithtoastkotlin/MainActivity.kt
package com.peace.helloworldwithtoastkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
//Kotlin automatically Binds Views with objects

class MainActivity : AppCompatActivity() {

    //private lateinit var btn: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //btn=findViewById(R.id.button)
        //here button is the button in the xml file
        button.setOnClickListener{
            //in Java we used editText.getText() now we simply use editText.text
            Toast.makeText(applicationContext,"Hello World From Kotlin "+editText.text,Toast.LENGTH_SHORT).show()
        }
    }

}
